app/models/category.rb

```diff
@@ -13,4 +13,8 @@
 class Category < ActiveRecord::Base
   validates_length_of :description, :maximum => 2000
   validates_presence_of :userid, :name
+
+  def self.link
```

Comment:
I think where possible we should be trying to use Rails's path helper methods instead of concatenating URIs all over the place... I don't think a model should know what (partial) path something else might need to display it (it's a separation of concerns thing). But this is several layers away from where we're at right now.